app/assets/javascripts/discourse/app/templates/components/sidebar/hamburger-dropdown.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="hamburger-panel">
+  <div class="hamburger-menu revamped menu-panel drop-down" data-max-width="320">
```

Comment:
I know it is v0 so this is just for the future. When channel is very long or no categories selected, I cannot pin sidebar anymore. We have to hide overflow content, but let's wait for chat mounted via API, that may already solve the issue
<img width="318" alt="Screen Shot 2022-07-22 at 1 38 48 pm" src="https://user-images.githubusercontent.com/72780/180357766-8e5690de-4fd4-4716-9bad-15bfefb170db.png">

Reply:
@awesomerobot I had to remove the width [here](https://github.com/discourse/discourse/pull/17600/commits/ee68023852dc2791216f539fb2cafcd8bd4bcfac) because it was causing this problem. We can continue to iterate after I merge the PR.